Library/Formula/libproxy.rb

```diff
@@ -0,0 +1,85 @@
+class Libproxy < Formula
+  desc "Library that provides automatic proxy configuration management"
+  homepage "https://libproxy.github.io/libproxy/"
+  url "https://github.com/libproxy/libproxy/archive/libproxy-0.4.11.tar.gz"
+  sha256 "af3525f9fdbe8d45ba4449ed1f02d01c21ad0bb5c31573e07f66be453c5f3a53"
+
+  stable do
+    # Fix Build on Mac OS X - Fixes bug 183 -- https://github.com/libproxy/libproxy/commit/25cc320452726924697de856e523c52acaa75645
+    # Fix compilation errors with clang -- https://github.com/andyetitmoves/libproxy/commit/baa844d5992c30e996ae7a552d7afee7099e8dfb
```

Comment:
I see it's been submitted: probably want to wait for it to be accepted/merged before we include this formula. Thanks!